backbone.js

```diff
@@ -1440,6 +1440,11 @@
       return path === this.root && !this.getSearch();
     },
 
+    // Decode unicode escapes without decoding `%25`.
```

Comment:
Updated in faea842. It's hard to strike a balance between succinctness and completeness so let me know if I should tweak it! :smiley: